src/quo/components/numbered_keyboard/keyboard_key/view.cljs

```diff
@@ -7,16 +7,23 @@
     [react-native.core :as rn]
     [reagent.core :as reagent]))
 
+(defn- label->accessibility-label
+  [label]
+  (let [label-name (if (keyword? label) (name label) label)]
```

Comment:
I think if condition is redundant. It can just be `(name label)`

Reply:
It's for handling number in keyboard which one calculates as integers not as strings - `:item      (+ (* (dec row-index) 3) column-index)`.
As an alternative, I can cast these numbers to strings and omit this condition

Reply:
@OmarBasem I have found another case - label could be `nil`

Reply:
In this case I can use  `(name (or label ''))`.
But also `keyboard-key/view` component won't work with an integer type.
It's ok?

Reply:
@Rende11 what makes label `nil`? Can this be avoided?

Reply:
@OmarBasem 
when `type` prop for `keyboard-key` component is `:derivation-path`, this label ignores and assumes that is unnecessary
https://github.com/status-im/status-mobile/blob/6470f90cf371a6e10c6016bc7ab9bec83af17414/src/quo/components/numbered_keyboard/keyboard_key/view.cljs#L41

Reply:
Ok fine, keep as it is